### PR TITLE
Fix test_all breakage

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Specify FUZZING_LANGUAGE

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Can confirm this fix works with helper.py

File: infra/helper.py [hunk 1]
```diff
@@ -595,10 +595,16 @@ def check_build(args):
       not _check_fuzzer_exists(args.project_name, args.fuzzer_name)):
     return 1
 
+  fuzzing_language = _get_project_language(args.project_name)
+  if fuzzing_language is None:
+    print('WARNING: language not specified in project.yaml. Defaulting to C++.')
```
Comment (inferno-chromium) [inline infra/helper.py:600]: Is this branch needed, i thought Max forced it to always exist.

**Review by inferno-chromium — APPROVED** [review]

File: infra/helper.py (hunk 1 above)
Comment (jonathanmetzman) [reply infra/helper.py:600]: No idea, but we do it anyway here: https://github.com/google/oss-fuzz/blob/master/infra/helper.py#L496

Comment (inferno-chromium) [reply infra/helper.py:600]: sgtm

**Comment by jonathanmetzman** [comment]
Fixes #4672

**Comment by jonathanmetzman** [comment]
Unfortunately I can't test that this fixes CIFuzz https://github.com/jonathanmetzman/cifuzz-example/runs/1431656506
until https://github.com/google/oss-fuzz/pull/4666 lands.

**Pull request merged**